src/main/java/edu/usf/cutr/OPC/FeedProcessor.java

```diff
@@ -150,20 +155,32 @@ else if (cause instanceof IndexOutOfBoundsException) {
                 System.out.println("\nGTFS Data Valid End Date: " + endTimestamp);
 		ResultSet rs = ps.executeQuery();
                 
+                sql = "UPDATE [" +database + "].[dbo].[vehicle_positions]\n" +
+                        "SET [closest_stop_id] = ?, [distance_to_stop] = ?, [schedule_deviation] = ?, [timepoint] = ? \n" +
+                        "WHERE [oid] = ?";
+                ps = conn.prepareStatement(sql);
+                long rt_arrivaltime;
 		while (rs.next()) {
                     String trip_id_rt = rs.getString("trip_id"); //gtfs-rt trip id
                     Double latVal = Double.parseDouble(Float.toString(rs.getFloat("position_latitude")));
                     Double lonVal = Double.parseDouble(Float.toString(rs.getFloat("position_longitude")));
-                    Timestamp timestamp = rs.getTimestamp("timestamp");
+                    Timestamp timestampUT = rs.getTimestamp("timestamp");
+                    Date timestampEST = new Date(timestampUT.getTime() + TimeZone.getTimeZone("EST").getRawOffset());
```

Comment:
Instead of hard-coding this to `EST`, could you pull this from the GTFS `agency.txt` `agency_timezone` field instead?